massgov/mayflower
# DP-12475 Remove the indentation feature part from richText.js

Any PRs being created needs a changelog.txt file before being merged into dev. See: [Change Log Instructions](https://github.com/massgov/mayflower/blob/dev/docs/change-log-instructions.md)


## Description

**TO-DO:  Set up something to maintain the same visual after removing richText.js.**

- Remove `richText.js` from the file system and `index.js` to replace the feature in Drupal.
- Remove js hook classes for indentation and anchor from `03-organisms/by-author/rich-text.twig`.
- Adjust rules-of-court.json's rich text output in sections to maintain the same visual presentation with the change.

## Related Issue / Ticket

- [\[BE + FE? Dev\] Convert Mayflower Patternlab's richtext.js into a Drupal text filter 2](https://jira.mass.gov/browse/DP-12475)
- [DP-12475 Convert Mayflower Patternlab's richtext.js into a Drupal text filter 2 #3040](https://github.com/massgov/mass/pull/3040)

## Steps to Test


1. In the file structure, check **richText.js** is removed from `patternlab/styleguide/source/assets/js/modules/` and in `patternlab/styleguide/source/assets/js/index.js`.

1. Go to the [Rules of court page](http://localhost:3000/?p=pages-rules-of-court).

1. Compare the section's indentations with the [current implementation](https://mayflower.digital.mass.gov/?p=pages-rules-of-court) for any difference.

## Screenshots
Use something like [licecap](http://www.cockos.com/licecap/) to capture gifs to demonstrate behaviors.


## Additional Notes:

Anything else to add?

#### Impacted Areas in Application


* 

#### @TODO


*

#### Today I learned...
